sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py

```diff
@@ -162,6 +173,24 @@ def __init__(self, **kwargs: Any) -> None:
 
             collect_statsbeat_metrics(self)
 
+        # Initialize customer statsbeat if enabled
+        self._customer_statsbeat_metrics = None
```

Comment:
Customers can define multiple exporters possibly with different instrumentation keys. We should probably support multiple instrumentation keys in the future but for now we can simply use the first one that we see that is instantiated. With that being said, `CustomerStatsbeatMetricsClass ` should then be a singleton so we don't instantiate it multiple times across exporters (there should be only one global instance of it that is keep track of stats metrics).